hooks/persistence-defectdojo/hook/src/main/java/io/securecodebox/persistence/config/EnvConfig.java

```diff
@@ -0,0 +1,79 @@
+package io.securecodebox.persistence.config;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * This class provides an API to access configuration values from environment variables
+ */
+@Slf4j
+public final class EnvConfig {
+
+  public boolean isDev() {
+    return Boolean.parseBoolean(retrieveEnvVar(EnvVarNames.IS_DEV));
+  }
+
+  public String scanName() {
+    return retrieveEnvVar(EnvVarNames.SCAN_NAME);
+  }
+
+  public String namespace() {
```

Comment:
@J12934 What's the purpose of this config?

Reply:
Its the kubernetes namespace the scan is running in.
Required for hooks to be able to update the status of the scan (e.g. if the changed the number of findings they should update the status of the scan (done automatically by the hook-sdk for all JS hooks (every hook but this one))
This is automatically set for every hook job